vhdbuilder/packer/vhd-image-builder-arm64-gen2.json

```diff
@@ -40,7 +40,7 @@
       "virtual_network_subnet_name": "{{user `subnet_name`}}",
       "ssh_read_write_timeout": "5m",
       "os_type": "Linux",
-      "os_disk_size_gb": 30,
+      "os_disk_size_gb": 60,
```

Comment:
again, unless we know for this change is okay for all SKUs, we should have a separate packer template that's GB200-specific